doc/api/errors.md

```diff
@@ -1116,6 +1116,14 @@ added:
 
 An attempt to invoke an unsupported crypto operation was made.
 
+<a id="ERR_CWD_DELETED"></a>
+
+### `ERR_CWD_DELETED`
+
+The current working directory has been deleted while the process was still inside it.
+
+To resolve this, use `process.chdir()` to switch to a valid directory.
```

Comment:
I'm not sure "while the process was still inside it" is very clear – also, `process.chdir()` does not exactly "resolve" it, the other dir could also be deleted before we call `process.cwd()`
```suggestion
The current working directory has been deleted during the process' lifetime.
```